tools/mysqlversion.py

```diff
@@ -39,12 +39,12 @@
 # Constraints for compatible MySQL/MariaDB versions
 _DB_CONSTRAINT = {
     _MYSQL: {
-        _MIN_VERSION: (5, 0),
-        _EXACT_VERSION: ['5.1.65', '5.1.73']
+        _MIN_VERSION: (8, 0),
```

Comment:
I think the minimum MySQL version mentioned by `README.md` and `tools/templates/sections.xml` needs to be updated as well.

Reply:
And we should probably mention that MariaDB works too.

Reply:
Good catch -- will do on both of these, thanks!